worlds/pokemon_emerald/docs/en_Pokemon Emerald.md

```diff
@@ -30,6 +30,7 @@ randomizer. Here are some of the more important ones:
 - The Wally catching tutorial is skipped
 - All text is instant and, with an option, can be automatically progressed by holding A
 - When a Repel runs out, you will be prompted to use another
+- Rematches with trainers have been disabled, even if the trainer is registered in the PokeNav
```

Comment:
Minor correction, but otherwise this LGTM
```suggestion
- Rematches with trainers have been disabled, even if the trainer is registered in the PokéNav
```